src/client/js/floatingButton.js

```diff
@@ -0,0 +1,37 @@
+import WrapImg from "../assets/image/floatingWrap.png"
+import ArrowImg from "../assets/image/floatingArrow.png"
+
+export default class FloatingButton {
+  #floatingButton = null;
+
+  constructor() {
+    const floatingButton = document.createElement('div');
+    
+    const wrap = document.createElement('img');
+    const arrow = document.createElement('img');
+    wrap.src = WrapImg;
+    arrow.src = ArrowImg;
+    arrow.className = "floatingArrow";
+
+    floatingButton.appendChild(wrap);
+    floatingButton.appendChild(arrow);
+    this.#floatingButton = floatingButton;
+
+    this.#floatingButton.addEventListener("click", event => {
+      if (event.currentTarget === this.$floatingButton) {
+        document.scrollingElement.scrollTo(0, 0);
+      }
+    })
+
+    document.querySelector("body").appendChild(this.#floatingButton);
+    // Refactor: 성능 개선
+    document.addEventListener('scroll', () => {
+      if(document.scrollingElement.scrollTop === 0) this.#floatingButton.style.visibility = 'hidden';
+      else this.#floatingButton.style.visibility = 'visible';
```

Comment:
애도 {} 로 묶어주시는 방법도 있으니 참고해주시면 좋을거 같습니다. 
```suggestion
      if(document.scrollingElement.scrollTop === 0) this.#floatingButton.style.visibility = 'hidden';
      else this.#floatingButton.style.visibility = 'visible';
```